src/components/download/DownloadDialog.js

```diff
@@ -149,7 +150,7 @@ export class DownloadDialog extends Component {
                 .getComputedStyle(titleEl, null)
                 .getPropertyValue('width');
 
-            titleEl.style.width = parseFloat(width) + 1 + 'px';
+            titleEl.style.width = parseFloat(width) + 2 + 'px';
```

Comment:
You could use string template here. Not crucial, but more standard these days.

Reply:
Fixed.